tests/test_tulok_bocc.py

```diff
@@ -0,0 +1,125 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.tulok_bocc import TulokBoccSpider
+
+# Load local JSON file for testing
+test_response = file_response(
+    join(dirname(__file__), "files", "tulok_bocc.json"),
+    url="https://tulsacook.api.civicclerk.com/v1/Events?$filter=categoryId+in+(26,40)",
+)
+
+# Load HTML file for testing info page parsing
+info_page_response = file_response(
+    join(dirname(__file__), "files", "tulok_bocc.html"),
+    url="https://www2.tulsacounty.org/Legacy/agendasdetail_civic.aspx?entity=BOCC",
+)
+
+spider = TulokBoccSpider()
+
+# Parse the info page first to extract time notes
+list(spider.parse_info_page(info_page_response))
+
+# Freeze time for consistent test results
+freezer = freeze_time("2025-12-02")
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
+
+freezer.stop()
+
+
+def test_first_item_properties():
+    item = parsed_items[0]
+    assert item["title"] == "Board of County Commissioners Regular Meeting"
+    assert item["start"] == datetime(2025, 12, 9, 9, 0)
+    assert item["end"] == datetime(2025, 12, 9, 11, 0)
+    assert item["status"] == "tentative"
+    assert item["location"] == {
+        "name": "Board of County Commissioners",
+        "address": "500 S Denver Ave Room 100 Tulsa, OK, 74103",
+    }
+    # Links
+    assert len(item["links"]) == 2
+    assert item["links"][0]["title"] == "Agenda"
+    assert item["links"][0]["href"].startswith("https://tulsacook.portal.civicclerk.com/")
```

Comment:
It is better to test the whole URL of the Agenda.

 for example:
```suggestion
    assert item["links"][0]["href"] == "https://tulsacook.portal.civicclerk.com/event/2182/files/agenda/9404"
```